README.md

```diff
@@ -276,12 +276,12 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 > 
 >     1     
  
-**PC_Envelope(p pcpatch)** returns **bytea**
+**PC_Envelope_AsBinary(p pcpatch)** returns **bytea**
```

Comment:
I am just suggesting to have `PC_Envelope` in the pointcloud-postgis extension, exactly the way you did in this PR. And if the user needs a WKB representation of the patch envelope then he will have to use `PC_Envelope` and PostGIS' `ST_AsBinary` function. So, no, no linking to the PostGIS lib.